addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -107,13 +107,18 @@
                     </RappresentanteFiscale>
                     <CessionarioCommittente>
                         <DatiAnagrafici>
-                            <IdFiscaleIVA t-if="record.commercial_partner_id.vat">
+                            <IdFiscaleIVA t-if="record.commercial_partner_id.vat and in_eu(record.commercial_partner_id)">
                                 <IdPaese t-esc="get_vat_country(record.commercial_partner_id.vat)"/>
                                 <IdCodice t-esc="get_vat_number(record.commercial_partner_id.vat)"/>
                             </IdFiscaleIVA>
-                            <IdFiscaleIVA t-if="not record.commercial_partner_id.vat and record.commercial_partner_id.country_id.code != 'IT'">
+                            <IdFiscaleIVA t-if="not record.commercial_partner_id.vat and record.commercial_partner_id.country_id.code != 'IT' and in_eu(record.commercial_partner_id)">
```

Comment:
I think the condition as before was correct as it should be the same for private clients as well.  I am also not sure it should have the IdPaese.  If there was no problem with that before.